.gitignore

```diff
@@ -7,6 +7,8 @@
 /OPT
 secrets.py
 
+.DS_Store
```

Comment:
Ideally, this shouldn't be part of a project's `.gitignore` file.  It should be in the local ignore file of the git client.  E.g., I use SourceTree and it uses my `~/.gitignore_global` file in addition to the project's own `.gitignore`.  That file includes `.DS_Store`.  If a git client is trying to commit that file, check the client's global ignore.